doc/eID/eid-integration-magnus.md

```diff
@@ -0,0 +1,37 @@
+# UseID Integration
+
+To integrate UseID and mobile app and web widget, the following steps are necessary for services already having an existing eID integration.
+
+## Integration
+
+### 1. Embedding Widget
+
+eService embeds widget via JavaScript by adding the following empty container to the HTML page:
+
+```html
+<div id="useid-widget-container" data-tc-token-url="{}"></div>
+```
+
+and including this script in the page:
+
+```html
+<script async src="https://useid.ds4g.net/widget.js"></script>
+```
+
+As value for `data-tc-token-url` the already known tcTokenURL is used.
+
+> Previously, here was a link including the tcTokenURL as query parameter for the user to click to open the app. Now a widget is displayed, which can be scanned by the user with the smartphone to start the eID scanning in the UseID mobile app.
+
+### 2. ID process
+
+The process of scanning the eID has not changed for the eService. Changes for the users are in using the UseID mobile app and their smartphone as scanning devices.
+
+### 3. Success and Refresh
+
+The user is returned to the `refreshAddress` , which is provided in the TC token, on the device displaying the widget.
```

Comment:
I would maybe add "The user is returned to the refreshAddress of the eService, ...